app/src/common/shared/com/igalia/wolvic/utils/StringUtils.java

```diff
@@ -89,4 +84,36 @@ public static String capitalize(@NonNull String input) {
             return input;
         }
     }
+
+    public static int compareVersions(@NonNull String version1, @NonNull String version2) {
+        if (version1.isEmpty() && version2.isEmpty())
+            return 0;
+        if (version1.isEmpty())
+            return -1;
+        if (version2.isEmpty())
+            return 1;
+
+        String[] components1 = version1.split("\\.");
+        String[] components2 = version2.split("\\.");
+        int len = Math.max(components1.length, components2.length);
+
+        for (int i = 0; i < len; i++) {
+            String component1 = i < components1.length ? components1[i] : "0";
```

Comment:
Nit: we can define this as a lambda, like `ToInt(component, i))` and then use it directly in the 'compare' call; something likes this: 

try {
   int comparison = Integer.compare(ToInt(component,  i), ToInt(component2, i));
}